content/en/tests/_index.md

```diff
@@ -71,8 +71,8 @@ In addition to tests, Test Visibility provides visibility over the whole testing
 | {{< ci-details title="Benchmark tests support" >}}Automatic detection of performance statistics for benchmark tests.{{< /ci-details >}}                                                                          | {{< X >}} |                      |                        | {{< X >}} |           | {{< X >}} |                        |
 | {{< ci-details title="Parameterized tests" >}}Automatic detection of parameterized tests.{{< /ci-details >}}                                                                                                     | {{< X >}} |       {{< X >}}      |       {{< X >}}        | {{< X >}} | {{< X >}} | {{< X >}} |                        |
 | {{< ci-details title="Early flake detection *" >}}Automatically retry new tests to detect flakiness.{{< /ci-details >}}                                                                                          | {{< X >}} |       {{< X >}}      |       {{< X >}}        |           |           |           |                        |
-| {{< ci-details title="Flaky test retries *" >}}Automatically retry failed flaky tests up to N times to avoid failing the build.{{< /ci-details >}}                                                               |           |       {{< X >}}      |                        |           |           |           |                        |
-| {{< ci-details title="Selenium RUM integration" >}}Automatically link browser sessions to test cases when testing RUM-instrumented applications.{{< /ci-details >}}                                            | {{< X >}} |       {{< X >}}      |       {{< X >}}        |           |           |           |                        |
+| {{< ci-details title="Auto test retries *" >}}Automatically retry failed tests up to N times to avoid failing the build due to tests flakiness.{{< /ci-details >}}                                               |           |       {{< X >}}      |                        |           |           |           |                        |
```

Comment:
Very "maybe-nit" thing:
```suggestion
| {{< ci-details title="Auto test retries *" >}}Automatically retry failed tests up to N times to avoid failing the build due to test flakiness.{{< /ci-details >}}                                               |           |       {{< X >}}      |                        |           |           |           |                        |
```